stable/airflow/templates/deployments-scheduler.yaml

```diff
@@ -101,6 +105,10 @@ spec:
               /usr/local/scripts/install-requirements.sh &&
               echo "executing initdb" &&
               airflow initdb &&
+            {{- if .Values.airflow.connections }}
```

Comment:
@antweiss Why is this included only in the block where `.Values.dags.initContainer.enabled` and `.Values.dags.initContainer.installRequirements` are true?

Reply:
i.e. I think it should be included also in the else block on line 122: https://github.com/helm/charts/pull/10317/files#diff-f4f5de6f12364c16a36eaea2622e0361R122

Reply:
PR to change the logic: #11495